app/WebParsing/Ligature.hs

```diff
@@ -0,0 +1,24 @@
+{- | Utilities for expanding ligatures in unicode text.
+
+     Currently this module expands the following ligatures:ﬀ,ﬁ,ﬂ,ﬃ,ﬄ,ﬅ,ﬆ
+-}
+module WebParsing.Ligature where
+
+import Prelude hiding (concatMap)
+
+import Data.Text.Lazy
```

Comment:
@RyanDJLee as we discussed in this meeting, you can change this import to `Data.Text`. There's no reason these functions should operate on Lazy text, when we're using it on strict `Text`.